AuroraEditor/Base/TabBar/UI/TabBarItem/TabBarItemIcon.swift

```diff
@@ -30,32 +30,22 @@ extension TabBarItem {
                 .lineLimit(1)
         }
         .frame(
-            // To horizontally max-out the given width area ONLY in native tab bar style.
             maxWidth: prefs.preferences.general.tabBarStyle == .native ? .infinity : nil,
-            // To max-out the parent (tab bar) area.
             maxHeight: .infinity
         )
         .padding(.horizontal, prefs.preferences.general.tabBarStyle == .native ? 28 : 23)
         .overlay {
             ZStack {
                 if isActive {
-                    // Close Tab Shortcut:
-                    // Using an invisible button to contain the keyboard shortcut is simply
-                    // because the keyboard shortcut has an unexpected bug when working with
-                    // custom buttonStyle. This is an workaround and it works as expected.
                     Button(
                         action: closeAction,
                         label: { EmptyView() }
                     )
                     .frame(width: 0, height: 0)
                     .padding(0)
                     .opacity(0)
-                    .keyboardShortcut("w", modifiers: [.command])
+                    .keyboardShortcut("w", modifiers: [.command, .option])
```

Comment:
Why command? in Xcode you close a tab with command + W not command option W

Reply:
Option click closes all tabs except current one

Reply:
Ok, this is breaking the normal Command + W, you should create a separate button for it.